### PR TITLE
Adds a box traitor item

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Adds a 1 TC syndicate box to badassery category.

# Wiki Documentation

Add this to wiki

# Changelog

:cl: SuperSlayer
rscadd: Box traitor item
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2269,3 +2269,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/stamp/syndiround
 	cost = 1
 	illegal_tech = FALSE
+
+/datum/uplink_item/badass/box
+	name = "Box"
+	desc = "If you are too lasy to find one."
```
Comment (redmoogle) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	desc = "If you are too lazy to find one."
```

**Comment by maxion12345** [comment]
Syndi stamp makes syndicate boxes you know

**Comment by ghost** [comment]
> Syndi stamp makes syndicate boxes you know

Yes

**Comment by Bibby0110** [comment]
this is way too bloaty imo, considering we already have ways to make them

**Pull request closed without merging**